docs/source/release_notes/release-notes.rst

```diff
@@ -83,6 +83,17 @@ develop
          - Reduced logging noise from large Cassandra gets and puts by removing ERROR messages and only providing stacktraces at DEBUG.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1590>`__)
 
+    *    - |new|
+         - Upon startup of a Cassandra-backed AtlasDB client with a
```

Comment:
Ah, okay. I guess it's an incentive to avoid overly long sentences as well, going to column 283 while not in and of itself wrong seems like a code (docs?) smell. I'll try to cut this down and/or split this a little.